ts/screens/PinLoginScreen.tsx

```diff
@@ -136,6 +134,12 @@ class PinLoginScreen extends React.Component<Props> {
           </Button>
           <View spacer={true} />
           <Text white={true}>{I18n.t("pin_login.pin.reset.tip")}</Text>
+          <View spacer={true} />
+          {this.props.inPayment ? (
+            <CancelPaymentLink navigation={this.props.navigation} />
```

Comment:
I don't think we need a new component here, `CancelPaymentLink` can be replaced by a transparent white button provided by React Native

Reply:
ok, as you wish :) 
Just a clarification. If the keyboard will be permanent on the pin screen, the button will be covered. Maybe a new access point to the cancel payment action can be considered?

Reply:
that's an interesting observation, @matteodesanti what do you think?

we should probably move the "cancel" action somewhere on top?